specification/signalr/data-plane/AzureSignalR/v1/azuresignalr.json

```diff
@@ -0,0 +1,605 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Azure SignalR Service REST API",
+    "version": "v1"
+  },
+  "paths": {
+    "/api/v1/health": {
+      "head": {
+        "tags": [
+          "general"
+        ],
+        "summary": "Get service health status.",
+        "operationId": "HealthApi_GetHealthStatus",
+        "responses": {
+          "200": {
+            "description": "The service is healthy"
+          }
+        }
+      }
+    },
+    "/api/v1/hubs/{hub}": {
+      "post": {
+        "tags": [
+          "signalr.v1"
+        ],
+        "summary": "Broadcast a message to all clients connected to target hub.",
+        "operationId": "ServiceApi_Broadcast",
+        "consumes": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "in": "path",
+            "name": "hub",
+            "description": "Target hub name, which should start with alphabetic characters and only contain alpha-numeric characters or underscore.",
+            "required": true,
+            "type": "string",
+            "pattern": "^[A-Za-z][A-Za-z0-9_`,.[\\]]{0,127}$"
+          },
+          {
+            "in": "query",
+            "name": "excluded",
+            "description": "Excluded connection Ids",
+            "type": "array",
+            "items": {
+              "type": "string"
+            },
+            "collectionFormat": "multi"
+          },
+          {
+            "in": "body",
+            "name": "payloadMessage",
+            "description": "The message body.",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/PayloadMessage"
+            }
+          }
+        ],
+        "responses": {
```

Comment:
Is there no response body in the case of failures? If there is, its schema should be included in the swagger. And even if there is no schema, I would include a `"default"` section explicitly calling out the fact that there is no response body.

Reply:
Quoted from @lmazuel , as the comment for my previous PR iteration which contains explicity error responses with `text/plain` body descriptions. The comment makes sense to me because they are not that informative and the error code to the most extent explains why it fails. So that's the reason I removed the error nodes.
> •	You preferred to use “x-ms-error-response” for the 400 errors that returns text/plain errors. Is there a specific reason why you chose to use the extension, and not just removing the status code completely from the Swagger? I know I gave you the two options, but I was slightly encouraging the “don’t say anything” approach. I don’t feel most of them are informative from a documentation perspective (it’s just say 400 – Bad request, which is at the HTTP spec level), and that makes you add “text/plain” in the “produces” node, which I’d rather avoid (since the success path does not return “text/plain”). So my advice would be more explicit about removing all those, and let azure-core in all languages deal with 4xx automatically.

Reply:
Why are you returning text/plain response bodies for errors? The Microsoft REST API Guidelines have a recommended json format for error responses that allow clients to programmatically understand what the error was. A 400 response basically only says "you did something wrong", which makes it more or less useless from a client perspective for anything other than logging it.

Reply:
Thanks for the insights! will follow https://github.com/microsoft/api-guidelines/blob/vNext/Guidelines.md#7102-error-condition-responses to add error response